index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <section>
+            <a class="logo">logos</a>
```

Comment:
- [ ] I have noticed that the previous reviewer suggested that you add a `<section>` here. However, I believe there is a misunderstanding. The over all structure could like something like this:
```
</body>
  <header>
    <a > Logo </a>
    <nav ...>.....</nav>
   </header
   <section class='headline'>
     <h1> YOUR NAME </h1>
     <h2> Software Developer </h2>
     <p> .....</p>
   </section>
   <section class='works'> ....</section>
   <section class='about-me'>....</section>
   ....
   <footer >....</footer>
</body>
```
Therefore, kindly adjust the skeleton accordingly.